hail/python/test/hail/fs/test_worker_driver_fs.py

```diff
@@ -14,7 +14,7 @@ def test_requester_pays_no_settings():
     try:
         hl.import_table('gs://hail-test-requester-pays-fds32/hello')
     except Exception as exc:
-        assert "Bucket is a requester pays bucket but no user project provided" in exc.args[0]
+        assert "Bucket is a requester pays bucket but no user project provided" in str(exc)
```

Comment:
these tests started failing with this change because the bucket's storage class is now checked before whatever code originally encountered the "no user project provided" error, which for some reason results in a different `Exception` subclass that doesn't have an `args` attribute, but it's still the same error message so i just changed the tests slightly

Reply:
Huh, weird. Thanks for the explanation!